requirements.txt

```diff
@@ -2,3 +2,5 @@ pytest-base-url
 pytest-selenium
 pytest-variables
 pytest-xdist
+pytest-testinfra
+apypie
```

Comment:
so it *was* a good decision to keep it separate and not fold it into FAM completely ;-)

Reply:
Yea, yea. :+1: